frontendActions/build.gradle

```diff
@@ -7,6 +7,8 @@ dependencies {
   implementation project(':branchLayoutApi')
   implementation project(':frontendActionIds')
   implementation project(':frontendDataKeys')
+  implementation project(':frontendGraphApi')
+  implementation project(':frontendUiTableImpl')
```

Comment:
:thinking: maybe expose more stuff to `frontendUiApi` instead? ofc not if that's gonna turn out too much engineering